db/migrate/20181206123910_create_foreman_tasks_links.foreman_tasks.rb

```diff
@@ -0,0 +1,28 @@
+class CreateForemanTasksLinks < ActiveRecord::Migration[4.2]
+  def change
+    # rubocop:disable Rails/CreateTableWithTimestamps
+    create_table :foreman_tasks_links do |t|
+      task_id_options = { :index => true, :null => true }
+      if on_postgresql?
+        t.uuid :task_id, task_id_options
+      else
+        t.string :task_id, task_id_options
+      end
+      # TODO: Figure out if we need this
+      # t.string :name, index: true, null: false
```

Comment:
Seems like something to decide

Reply:
I can't think of a reason why we should keep this around, I'd drop it.

Reply:
:+1: